cockroachcloud/serverless-movr.md

```diff
@@ -0,0 +1,125 @@
+  {% include copy-clipboard.html %}
+  ~~~ shell
+  cockroach workload run movr \
+  --duration=1m \
+  'postgresql://<username>:<password>@serverless-host>:26257/movr?sslmode=verify-full&options=--cluster%3D<routing-id>'
+  ~~~
+
+  You'll see per-operation statistics print to standard output every second:
+
+  ~~~
+  _elapsed___errors__ops/sec(inst)___ops/sec(cum)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)
+      1.0s        0           31.9           32.0      0.5      0.6      1.4      1.4 addUser
+      1.0s        0            6.0            6.0      1.2      1.4      1.4      1.4 addVehicle
+      1.0s        0           10.0           10.0      2.2      6.3      6.3      6.3 applyPromoCode
+      1.0s        0            2.0            2.0      0.5      0.6      0.6      0.6 createPromoCode
+      1.0s        0            9.0            9.0      0.9      1.6      1.6      1.6 endRide
+      1.0s        0         1407.5         1407.8      0.3      0.5      0.7      4.1 readVehicles
+      1.0s        0           27.0           27.0      2.1      3.1      4.7      4.7 startRide
+      1.0s        0           86.8           86.9      4.7      8.4     11.5     15.2 updateActiveRides
+      2.0s        0           26.0           29.0      0.5      1.1      1.4      1.4 addUser
+      2.0s        0            8.0            7.0      1.2      2.8      2.8      2.8 addVehicle
+      2.0s        0            2.0            6.0      2.6      2.8      2.8      2.8 applyPromoCode
+      2.0s        0            0.0            1.0      0.0      0.0      0.0      0.0 createPromoCode
+      2.0s        0            6.0            7.5      0.8      1.7      1.7      1.7 endRide
+      2.0s        0         1450.4         1429.1      0.3      0.6      0.9      2.6 readVehicles
+      2.0s        0           17.0           22.0      2.1      3.3      5.5      5.5 startRide
+      2.0s        0           59.0           72.9      6.3     11.5     11.5     14.2 updateActiveRides
+  ...
+  ~~~
+
+  After the specified duration (1 minute in this case), the workload will stop and you'll see totals printed to standard output:
+
+  ~~~
+  _elapsed___errors_____ops(total)___ops/sec(cum)__avg(ms)__p50(ms)__p95(ms)__p99(ms)_pMax(ms)__result
+     60.0s        0          85297         1421.6      0.7      0.3      2.6      7.1     30.4
+  ~~~
+      
+## Step 5. Calculate your budget
+
+1. In the {{ site.data.products.db }} Console, close the **Connect to cluster** dialog to return to your cluster's [**Overview** page](cluster-overview-page.html).
+
+1. In the **Usage this month** section, note the number of RUs your cluster used. You can also see the **Request Units** graph for a breakdown of how many RUs per second you were using over the last minute.
+
+1. Multiply your usage over one minute by the number of minutes in a month (43800) to estimate how many RUs the workload would use in a month. For example, if you used 1800 RUs: 1800 RUs/min * 43800 min = 78,840,000 RUs.
```

Comment:
depending on the reader it might be useful to update this a little to show the math.  the only reason I'd even say anything is because I did `60*24*31 = 44640`

in order to get near 43800 I had to do something fussy like `60*24*30.4166666667 = 43800.000000048`

which is probably correct but esp since this is about $$$ I think it would be better to have a simpler minutes estimate so it's easy for the user to replicate using e.g `python3 -c 'print(60*24*31)'` (or 30.5 or whatever, as long as it's easy to replicate)

easy to replicate the number = easier to trust

or you could say something about how there's a fudge factor and the real number is 43800 or whatever

however since we're making up the 1800 RUs anyway we might as well fudge the minutes to make the calculation easier to replicate for the user IMO